hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/OMClientRequest.java

```diff
@@ -181,17 +181,6 @@ protected OMResponse createErrorOMResponse(OMResponse.Builder omResponse,
     return omResponse.build();
   }
 
-
-  /*
-   * This method sets the omRequest. This will method will be called when
-   * preExecute modifies the original request.
-   * @param updatedOMRequest
-   */
-  protected void setUpdatedOmRequest(OMRequest updatedOMRequest) {
-    Preconditions.checkNotNull(updatedOMRequest);
-    this.omRequest = updatedOMRequest;
-  }
```

Comment:
We are not updating omRequest in OmClientRequest anymore?

Reply:
Yes that is the reason i reemoved this method. this is done based on the above comment execute. Will revisit during merge HA/Non-HA code.